go/vt/worker/vertical_split_diff.go

```diff
@@ -448,3 +452,9 @@ func (vsdw *VerticalSplitDiffWorker) diff(ctx context.Context) error {
 
 	return rec.Error()
 }
+
+// markAsFailed records the error and changes the state of the worker to reflect this
+func (vsdw *VerticalSplitDiffWorker) markAsFailed(recorder concurrency.ErrorRecorder, err error) {
```

Comment:
nit: I suggest to rename "recorder" to "er" to be consistent with the rest of the code base.